docs/sources/datasources/google-cloud-monitoring/google-authentication.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Authentication"
+description = "Google authentication"
+keywords = ["grafana", "google", "authentication"]
+aliases = ["/docs/grafana/next/datasources/cloudmonitoring/"]
+weight = 5
++++
+
+# Google authentication
+
+Requests from a Grafana plugin to Google are made on behalf of an IAM role or an IAM user. The IAM user or IAM role must have the associated policies to perform certain API actions. Since these policies are specific to each data source, refer to the data source documentation for details.
+
+All requests to Google APIs are performed on the server side by the Grafana backend.
```

Comment:
```suggestion
Requests from a Grafana plugin to Google are made on behalf of an IAM role or an IAM user. The IAM user or IAM role must have the associated policies to perform certain API actions. Since these policies are specific to each data source, refer to the data source documentation for details. All requests to Google APIs are performed on the server-side by the Grafana backend.
```